e2e/claims.service.e2e.ts

```diff
@@ -825,109 +825,103 @@ describe('Сlaim tests', () => {
         expirationDate,
       });
     };
-
-    test('verifyVc should verify a VC with no errors if the issuer is authorized', async () => {
-      await signerService.connect(rootOwner, ProviderType.PrivateKey);
-      await domainsService.createRole({
-        roleName: verifyVcRole,
-        namespace,
-        data: roles[`${verifyVcRole}.${root}`],
-        returnSteps: false,
-      });
-      const vc = await createExampleSignedCredential(
-        [],
-        `${verifyVcRole}.${root}`
-      );
-      const result = await claimsService.verifyVc(vc);
-      expect(result.errors).toHaveLength(0);
-      expect(result.isVerified).toBeTruthy;
-    });
-    test('resolveCredentialAndVerify should resolve a Verifiable Credential and call correct verification method (verifyVC)', async () => {
-      const roleName = `${resolveVC}.${root}`;
-      await signerService.connect(rootOwner, ProviderType.PrivateKey);
-      await domainsService.createRole({
-        roleName: resolveVC,
-        namespace,
-        data: roles[roleName],
-        returnSteps: false,
-      });
-      await signerService.connect(staticIssuer, ProviderType.PrivateKey);
-      await createExampleSignedCredential([], roleName);
-      const { issuedToken } = await enrolAndIssue(rootOwner, staticIssuer, {
-        subjectDID: rootOwnerDID,
-        claimType: roleName,
-        registrationTypes: [
-          RegistrationTypes.OnChain,
-          RegistrationTypes.OffChain, // role type issuer should have offchain claim
-        ],
-        publishOnChain: true,
-        issuerFields: [],
-      });
-      await signerService.connect(rootOwner, ProviderType.PrivateKey);
-      let subjectDoc = await didRegistry.getDidDocument({
-        did: rootOwnerDID,
-        includeClaims: true,
-      });
-      mockCachedDocument.mockResolvedValueOnce(subjectDoc);
-
-      await claimsService.publishPublicClaim({
-        claim: { token: issuedToken },
+    describe('Verify off-chain claim and credential', () => {
+      test('verifyVc should verify a VC with no errors if the issuer is authorized', async () => {
+        await signerService.connect(rootOwner, ProviderType.PrivateKey);
+        const vc = await createExampleSignedCredential(
+          [],
+          `${verifyVcRole}.${root}`
+        );
+        const result = await claimsService.verifyVc(vc);
+        expect(result.errors).toHaveLength(0);
+        expect(result.isVerified).toBeTruthy;
       });
+      test('resolveCredentialAndVerify should resolve a Verifiable Credential and call correct verification method (verifyVC)', async () => {
+        const roleName = `${resolveVC}.${root}`;
+        await signerService.connect(rootOwner, ProviderType.PrivateKey);
+        await signerService.connect(staticIssuer, ProviderType.PrivateKey);
+        await createExampleSignedCredential([], roleName);
```

Comment:
Is this needed? It seems the return result isn't being assigned to a variable.

Perhaps there is a side-effect, however if that is the case, I think it is better for functions with side-effects to have a `void` return as this suggests to the reader that the function performs a side-effect (as otherwise, the void-return function wouldn't be useful).
Maybe this is old hat to you, but in other words, in clean code, IMO, there should be two types of functions:
- `pureFunction(input1, input2): returnType`
- `functionWithSideEffects(input1): void`